apps/pdal.cpp

```diff
@@ -46,6 +46,13 @@
 #include <string>
```

Comment:
No, my first instinct was just to register it right away.  If that's a more reasonable spot it can certainly be moved there.